exercises/clock/README.md

```diff
@@ -8,8 +8,7 @@ Two clocks that represent the same time should be equal to each other.
 
 ## Hints
 This exercise requires you to implement a type-specific method for determining equality of instances.
-For more information, see [this page]
-(https://docs.microsoft.com/en-us/dotnet/core/api/System.IEquatable-1).
+For more information, see [this page](https://docs.microsoft.com/en-us/dotnet/core/api/System.IEquatable-1).
```

Comment:
```suggestion
For more information, see [this page](https://docs.microsoft.com/en-us/dotnet/api/System.IEquatable-1?view=netcore-2.1).
```

Maybe we can directly link to the .NET Core 2.1 version?

Reply:
Sounds good!